views/css/ps_socialfollow.css

```
@@ -0,0 +1,5 @@
+.discord {
```

@marsaldev this class is too generic, let's go with the, maybe:

```suggestion
.ps-socialfollow-discord {
```

cc @NeOMakinG

---

I thought about it 😅 but I have aligned the name to old classes.
But I agree with you 👍🏻

---

Agree